woof-code/packages-templates/bash_FIXUPHACK

```diff
@@ -1,6 +1,6 @@
 DISABLE_POST_INSTALL_SCRIPT=yes
 
-if [ "$DISTRO_BINARY_COMPAT" = "void" ]; then
+if [ -f usr/bin/bash && ! -e bin/bash -a ! -L bin/bash ] ; then
```

Comment:
Shouldn't that be `[ -f usr/bin/bash -a ! -e bin/bash -a ! -L bin/bash ]`, or even better `[ -f usr/bin/bash ] && [ ! -e bin/bash ] && [ ! -L bin/bash ]`

Reply:
Yep